### ISSUE TITLE
[Bug] Unexpected MODIFIER at 35, expected END

### ISSUE THREAD

**Issue by xero88** [nutboltu/storybook-addon-mock#36]
I'm trying to mock, but i got error 

`Unexpected MODIFIER at 35, expected END`

```
Call Stack
 mustConsume
  vendors~main.iframe.bundle.js:1634:15
 parse
  vendors~main.iframe.bundle.js:1693:20
 stringToRegexp
  vendors~main.iframe.bundle.js:1846:27
 pathToRegexp
  vendors~main.iframe.bundle.js:1919:12
 match
  vendors~main.iframe.bundle.js:1766:26
 Faker/<
  vendors~main.iframe.bundle.js:2101:37
 Faker/<
  vendors~main.iframe.bundle.js:2132:26
 send/<
  vendors~main.iframe.bundle.js:1482:33

```

Here the code : 

```
import { SectorBlueFilter } from 'core/components/BlueFilter/SectorBlueFilter'
import withMock from 'storybook-addon-mock'

export default {
  title: 'Ui/Form/sectorbluefilter/SectorBlueFilter',
  component: SectorBlueFilter,
  decorators: [withMock]
}

export const Default = args => <SectorBlueFilter {...args} />

Default.args = {}
Default.argTypes = {
  onChangeOption: { action: 'site / sector changed' }
}
Default.parameters = {
  mockData: [
    {
      url: 'http://localhost:10510/brain/sites/sectors?all=true',
      method: 'GET',
      status: 200,
      response: [
        {
          id: 8,
          name: 'Demo'
        }
      ]
    }
  ]
}
```

**Comment by nutboltu** [comment]
@xero88 
This happens due to the port number in the URL. The faker method failed to parse URLs with the port number. I have labelled it as a bug. The fix will be in place soon.

**Comment by nutboltu** [comment]
@xero88 This issue is fixed in version `1.1.0`. Closing this issue.
